service/src/main/resources/application.yaml

```diff
@@ -59,3 +59,50 @@ fpl:
     mapping: 'example=>cafcass:cafcass@cafcass.com'
   local_authority_user:
     mapping: 'example=>1,2,3'
+
+---
+
+spring:
+  profiles: local
+
+
+management:
+  endpoints:
+    enabled-by-default: false
+
+idam:
+  api:
+    url: http://localhost:4501
+  s2s-auth:
+    url: http://localhost:4502
+    totp_secret: AABBCCDDEEFFGGHH
+
+core_case_data:
+  api:
+    url: http://localhost:4452
+
+document_management:
+  url: http://localhost:4506
+
+notify:
+  api_key: fake-key
+
+ccd:
+  ui:
+    base:
+      url: http://localhost:3451
+
+appinsights:
+  instrumentationkey: fake-key
+
+fpl:
+  local_authority_email_to_code:
+    mapping: 'swansea.gov.uk=>SA;hillingdon.gov.uk=>HN'
+  local_authority_code_to_name:
+    mapping: 'SA=>Swansea City Council;HN=>London Borough Hillingdon;PCC=>Portsmouth City Council;STF=>Staffordshire County Council;SCC=>Southampton City Council'
+  local_authority_code_to_hmcts_court:
+    mapping: 'SA=>Swansea Family Court:FamilyPublicLaw+sa@gmail.com;HN=>Portsmouth Combined Court:FamilyPublicLaw+hn@gmail.com'
+  local_authority_code_to_cafcass:
+    mapping: 'SA=>Cafcass Cymru:FamilyPublicLaw+cafcassWales@gmail.com;HN=>Cafcass:FamilyPublicLaw+cafcassEngland@gmail.com'
+  local_authority_user:
+    mapping: 'SA=>31,32,33;HN=>34,35'
```

Comment:
Just a reminder that having two ways of running application will increase maintenance overhead. I feel it still is worth running it on host side to save time, I am just conscious of cons.

Reply:
Ok, I see your point, let's see how it goes. 

Maybe we could create common part for something like fpl config in application.yml and use it from docker environment. Then only modify the IP addresses in env variables. I will check this as the next step if you don't mind!